src/Form/AdminSettingsForm.php

```diff
@@ -317,6 +318,31 @@ private function buildThumbnailUrlForm() {
   }
 
   /**
+   * Display advanced configurations forms.
+   *
+   * @return array
+   *   The render array for the advanced configuration form.
+   */
+  private function buildAdvancedConfigurationForm() {
+    $settings = $this->config('acquia_lift.settings');
+
+    $form = [
+      '#title' => t('Advanced configuration'),
+      '#type' => 'details',
+      '#tree' => TRUE,
+      '#open' => FALSE,
+    ];
+    $form['content_replacement_mode'] = [
+      '#type' => 'radios',
+      '#title' => t('Content replacement mode'),
+      '#default_value' => $settings->get('content_replacement_mode'),
+      '#options' => ['trusted' => t('Trusted'), 'untrusted' => t('Untrusted')],
+    ];
+
+    return $form;
+  }
```

Comment:
OK - so I'm still very much not fond of the term "deployment" and I think using a "Content" heading would be even more confusing as everything in the "Data Collection" grouping appears to have more to do with content than the replacement mode does.  It is more of an Advanced setting because we will have a default value (currently untrusted as it is what we have tested most - but likely to change to trusted).  The sysadmin would only visit this section if they had a specific use-case to change the functionality for the whole site, hence "advanced configuration".  Overriding on a per-slot basis is extremely advanced use-case that really belongs more in documentation and less as an in-your-face additional possible layer of confusion.

There are definitely plans for descriptions, however, it was intended to add these in the d8 module refinement story as we have a better handle on use cases, etc. to include and/or (even better) a link to documentation.

@tottey and I had discussed this prior to my inclusion of the option in the admin (as the original story only requires it being set by drush config value).